script/install_clang.sh

```diff
@@ -37,6 +37,9 @@ then
     then
         # llvm-symbolizer is required for meaningful output. This is part of the llvm base package which we don't install by default.
         travis_retry sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install llvm-${ALPAKA_CI_CLANG_VER}
+
+        # The sanitizer libraries are part of libclang-rt-${ALPAKA_CI_CLANG_VER}-dev
+        travis_retry sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install libclang-rt-${ALPAKA_CI_CLANG_VER}-dev
```

Comment:
This does not seem to work as intended :-(
```
+++++ sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install libclang-rt-14-dev
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  gcc-9-base libasan5 libatomic1 libc-dev-bin libc6-dev libclang-cpp14
  libclang1-14 libcrypt-dev libgc1c2 libgcc-9-dev libgomp1 libitm1 liblsan0
  libobjc-9-dev libobjc4 libquadmath0 libstdc++-9-dev libtsan0 libubsan1
  linux-libc-dev
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  lib32gcc-s1 lib32stdc++6 libc6-i386
The following packages will be REMOVED:
  clang-14 libclang-common-14-dev
The following NEW packages will be installed:
  lib32gcc-s1 lib32stdc++6 libc6-i386 libclang-rt-14-dev
0 upgraded, 4 newly installed, 2 to remove and 0 not upgraded.
```

Reply:
Yeah, looks like a packaging error on LLVM's side. I will investigate.

Reply:
Someone already opened a bug report on LLVM's tracker about this: https://github.com/llvm/llvm-project/issues/59903

Reply:
Since this probably won't be a fast fix on LLVM's side I suggest we fix this by adding clang-15 support. A quick test on my local Ubuntu 22.10 showed that the installation of `libclang-rt-14-dev` works fine.